server/src/test/copy_move.test.ts

```diff
@@ -987,3 +989,164 @@ test("cannot copy/move into assigned problem set or question bank or move out",
     }),
   ).rejects.toThrow("Cannot move content in an assigned activity");
 });
+
+test("MoveCopyContent allowed parent types", async () => {
```

Comment:
It wasn't clear to me from the test title what this test was doing. I see now that it is testing that `singleDoc` is not an allowed parent type.